app/views/judgmentDashboard.scala.html

```diff
@@ -1,14 +1,23 @@
-@()(implicit request: RequestHeader, messages: Messages)
-@main("Judgment Transfer Welcome") {
-    @defining(play.core.PlayVersion.current) { version =>
+@import views.html.helper.CSRF
+@import views.html.helper.form
+@()(implicit messages: Messages, request: RequestHeader)
 
-        <div class="govuk-grid-row">
-            <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-l">Welcome to the Transfer Digital Records service</h1>
+@main("Welcome") {
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Welcome to the Transfer Digital Records service</h1>
 
-                <h2 class="govuk-heading-m govuk-!-margin-top-9">Upload your judgment to start a new transfer</h2>
-                <p class="govuk-body">Placeholder page</p>
-            </div>
+      <h2 class="govuk-heading-m govuk-!-margin-top-9">Upload your records to start a new transfer</h2>
```

Comment:
Not sure what the designs say, but think this should actually say: _Upload your **judgment** to start a new transfer_

Reply:
The designs do say records but looking at the designs for all the other tickets, they all say judgment so it looks like it should be judgment. I've changed it. I also had to change the e2e tests back so can you re-check this one https://github.com/nationalarchives/tdr-e2e-tests/pull/139 when you get a second.